packages/modules/cache/src/utils/strategy.ts

```diff
@@ -0,0 +1,159 @@
+import {
```

Comment:
self-review: add onapplication* hooks and move subscriber creation to onapplicationstart